recipes/implicit/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "implicit" %}
+{% set version = "0.2.7" %}
+{% set sha256 = "b8c5fd270a3335988d8cff33f6e31b721d6d13813a02e6f61e21de1d62526267" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not x86_64]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython >=0.24
+    - gcc  # [linux or osx]
+    - numpy 1.10.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
+    - scipy >=0.16
```

Comment:
I think this can be removed from `build` requirements (because of `--single-version-externally-managed`). Probably a good idea, just to double-check that we're not linking against any scipy libraries (in which case we'd have to be more careful about version pinning).